libs/net/dal/Migrations/Initial/Up/PostUp/02-DataSourceSchedule.sql

```diff
@@ -77,4 +78,28 @@ IF (CASTANET_ID IS NOT NULL) THEN
   );
 END IF;
 
+IF (CPNEWS_ID IS NOT NULL) THEN
+  INSERT INTO public.data_source_schedule (
+    "data_source_id"
+    , "schedule_id"
+    , "created_by_id"
+    , "created_by"
+    , "created_on"
+    , "updated_by_id"
+    , "updated_by"
+    , "updated_on"
+    , "version"
+  ) VALUES (
+    CPNEWS_ID  -- data_source_id
+    , 3  -- schedule_id
```

Comment:
We don't want multiple data sources sharing the same schedule presently.  While we do enable many-to-many for future possibilities, presently when an admin edits a schedule it's for the specific data source.  Make sure there is a schedule for each data source you configure.

Reply:
Updated, thanks